test/install.bats

```diff
@@ -88,3 +88,13 @@ load helpers
   [ "$lines" = "1.0.0 is already installed." ]
   [ ! -r "$SWIFTENV_ROOT/version" ]
 }
+
+@test "does't allow --user on macOS" {
```

Comment:
I'm confused, isn't `--user` _only_ supported on macOS?

Reply:
You're right this test title is the wrong way round. It should read "doesn't allow --no-user on platforms other than macOS" or similiar, I'll update.